esignet-service/internal/config/app.go

```
@@ -32,7 +32,9 @@ const (
 	defaultDesignCacheTTLSecs    = 86400
 	defaultFlowCacheTTLSecs      = 86400
 
-	defaultHTTPClientTimeoutSecs         = 30
+	defaultMetricsPort = 9090
+
+	defaultHTTPClientTimeoutSecs = 30
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)